account_constraints/model/account_bank_statement.py

```diff
@@ -62,3 +62,14 @@ def process_reconciliation(self, mv_line_dicts):
         self = self.with_context(from_parent_object=True)
         return super(AccountBankStatementLine, self)\
             .process_reconciliation(mv_line_dicts)
+
+    @api.multi
+    def _is_account_cancel_installed(self):
+        ir_module = self.env['ir.module.module']
+        account_cancel = ir_module.search([('name', '=', 'account_cancel'),
+                                           ('state', '=', 'installed')])
+        return bool(account_cancel)
```

Comment:
you could use: 

    return ir_module.search_count([('name', '=', 'account_cancel'),
                                                        ('state', '=', 'installed')])

(replaces 69->71)